### PR TITLE
feat: seda extension.

### PR DESCRIPTION
**Pull request by davsclaus**

### CODE REVIEW

File: integration-tests/pom.xml [hunk 1]
```diff
@@ -147,6 +147,7 @@
         <module>validator</module>
         <module>xslt</module>
         <module>zipfile</module>
+        <module>seda</module>
```
Comment (ppalaga) [inline integration-tests/pom.xml]: `mvn process-resources -Pformat` would sort this one too.

File: extensions/pom.xml [hunk 2]
```diff
@@ -92,6 +92,7 @@
         <module>vm</module>
         <module>xslt</module>
         <module>zipfile</module>
+        <module>seda</module>
```
[the remark this answers is not in the document]
Comment (davsclaus) [reply extensions/pom.xml]: I did that but its not reordering them, from the root folder. Just ran it again and no files are changed

**Comment by lburgazzoli** [comment]
ok to test

File: extensions/pom.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ppalaga) [reply extensions/pom.xml]: Good to know. Let me check what is happening.

**Comment by oscerd** [comment]
ok to test

**Pull request merged**